src/main/scala/edu/ie3/powerFactory2psdm/converter/GridConverter.scala

```diff
@@ -8,28 +8,26 @@ package edu.ie3.powerFactory2psdm.converter
 
 import edu.ie3.datamodel.models.input.NodeInput
 import edu.ie3.powerFactory2psdm.model.Subnet
-import edu.ie3.powerFactory2psdm.model.powerfactory.{
-  GridModel,
-  Node,
-  RawGridModel
-}
+import edu.ie3.powerFactory2psdm.model.powerfactory.{GridModel, RawGridModel}
 
 /**
   * Functionalities to transform an exported and then parsed PowerFactory grid to the PSDM.
   */
 case object GridConverter {
 
   def convert(pfGrid: RawGridModel) = {
-    val gridElements = convertGridElements(pfGrid)
+    val grid = GridModel.build(pfGrid)
+    val gridElements = convertGridElements(grid)
   }
 
   /**
     * Converts the grid elements of the PowerFactory grid
     *
     * @param rawGrid the raw parsed PowerFactoryGrid
     */
-  def convertGridElements(rawGrid: RawGridModel): Unit = {
-    val grid = GridModel.build(rawGrid)
+  def convertGridElements(
+      grid: GridModel
```

Comment:
I would like to ask you to sort out stuff around the `GridModel`. Currently, there are three models out there. The `RawGridModel`, the `GridModel` and the `GridContainer`s from PowerSystemDataModel. What is the purpose of each and can they be named in a way, that is more distinctive?